### PR TITLE
(feat:gatsby-plugin): add support for aphrodite #8709

### PR DESCRIPTION
**Pull request by thisizkp**
Related to #8709 

This is my first contribution to gatsby and also first gatsby plugin, please do let me know if any further feedback. Haven't published the npm package yet, will do once the code review is done. 

Thanks a lot in advance for the support :)

### CODE REVIEW

**Comment by thisizkp** [comment]
sure @DSchau

File: packages/gatsby-plugin-aphrodite/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2"
+  },
+  "scripts": {
+    "build": "babel src --out-dir . --ignore **/__tests__",
+    "prepare": "npm run build",
+    "watch": "babel -w src --out-dir . --ignore **/__tests__"
+  },
+  "peerDependencies": {
+    "aphrodite": "^2.2.3",
```
Comment (DSchau) [inline packages/gatsby-plugin-aphrodite/package.json]: Nit: but maybe just use ^2.0.0 so we're not _too_ high here.

Comment (DSchau) [reply packages/gatsby-plugin-aphrodite/package.json]: Updated it myself :) Going to merge/publish now, thanks so much!

**Comment by DSchau** [comment]
@thisizkp thanks so much for the contribution. A miss on our part, but unfortunately, it looks like this plugin [already exists](https://www.npmjs.com/package/gatsby-plugin-aphrodite) so I'll revert this commit.

Still, we sincerely appreciate your contribution here, so enjoy your swag :)

**Comment by thisizkp** [comment]
@DSchau it's okay. Thanks for the support. Good learning experience.

**Pull request merged**